client/landing/stepper/declarative-flow/internals/steps-repository/setup-blog/index.tsx

```diff
@@ -35,6 +35,14 @@ const SetupBlog: Step = ( { navigation } ) => {
 
 	const { siteTitle, setComponentSiteTitle, tagline, setTagline } = useSetupFormInitialValues();
 
+	useEffect( () => {
+		// Clear site title and show placeholder for the flows below
+		const isStartWritingFlow = 'start-writing' === flow;
+		if ( isStartWritingFlow ) {
```

Comment:
Should we use `isStartWritingFlow( flow )` instead?

Reply:
Changed [here](https://github.com/Automattic/wp-calypso/pull/76711/commits/3fcea6fae0234aee0be261ed80792a01aa944e76)